crates/bevy_math/src/primitives/mod.rs

```diff
@@ -0,0 +1,13 @@
+//! This module defines primitive shapes.
+//! The origin of all primitives is at 0,0(,0) unless stated otherwise.
```

Comment:
```suggestion
//! The origin of all primitives is at (0, 0, 0) unless stated otherwise.
```

Reply:
Not entirely what I was going for, since the last (,0) is because of 2d vs 3d, but I definitely agree it should be changed. Just not sure what would be clear for both 2d and 3d

Reply:
I'd just do (0, 0, 0) since in practise it works for both dimensions and should still be very clear.

Alternatively, mention both:
```suggestion
//! The origin of all primitives is (0, 0) for 2D and (0, 0, 0) for 3D unless stated otherwise.
```
which is a bit more verbose, or refer to it as the "geometric origin"
```suggestion
//! The origin of all primitives is at the geometric origin unless stated otherwise.
```
but maybe that's more ambiguous?

Reply:
GitHub isn't playing nice right now... It duplicated my message since I had a pending review, but it was still treated as one comment so deleting the duplicate deleted both

What I said was that I'd do just (0, 0, 0) since it's clear and in practise it works for both dimensions even if Z doesn't technically exist in 2D.

Alternatively, mention both:
```suggestion
//! The origin is (0, 0) for 2D primitives and (0, 0, 0) for 3D primitives unless stated otherwise.
```
or describe it as the "geometric origin" or center:
```suggestion
//! The origin of all primitives is at the geometric origin unless stated otherwise.
```
but that might be slightly more ambiguous?